app/javascript/packages/masked-text-toggle/index.js

```diff
@@ -0,0 +1,29 @@
+class MaskedTextToggle {
```

Comment:
I'd kinda hoped to be able to skip JavaScript altogether for this and use [a `input:checked ~ .sibling` trick](https://codepen.io/aduth/pen/RwVeboy), but given the arrangement of the checkbox relative to (after) the masked text, it doesn't seem feasible without resorting to inaccessible options like [`order`](https://developer.mozilla.org/en-US/docs/Web/CSS/order).